elf_version.h

```diff
@@ -0,0 +1,132 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/**
+ *
+ * <H2> Version Number Semantics </H2>
+ *
+ * The version number is a sequence of four numbers, generally separated by dots
+ * when written. These are, in order, the Major number, the Minor number, the
+ * Revision number, and the Mission Revision number. At their option, Missions
+ * may modify the Mission Revision information as needed to suit their needs.
+ *
+ * It is important to note that Major, Minor, and Revision numbers are only
+ * updated upon official releases of tagged versions, \b NOT on development
+ * builds.
+ *
+ * The MAJOR number shall be incremented on release to indicate when there is a
+ * change to an API that may cause existing, correctly-written cFS components to
```

Comment:
I recommend not repeating this documentation.  Just reference where it's defined (once) if needed.  Too many times our documentation diverges, and or makes the doxygen confusing.